packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -324,6 +324,7 @@ export const raiseHandButtonSelector: RaiseHandButtonSelector;
 export type ReactionButtonSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
     checked?: boolean;
     disabled?: boolean;
+    allowed?: boolean;
```

Comment:
Looking for a better suggestion for this property name. 
This selector property reflects the meeting organizer policy on sending/receiving Reaction Capability in a teams interop meeting.

Reply:
As this is just used for the composite, remove from the component bindings. As a rule of thumb, anything in the selector should map directly to a prop on the react-components' interface

Reply:
Instead you can create a new selector under callcomposite/selectors that can retrieve this property

Reply:
Got it. Thanks :)

Reply:
So I've refactored and removed the `allowed` property and created a new selector in react-composite package to decide the hide/show of the button in the control bar according to capability.